ruby.rb

```diff
@@ -5,46 +5,86 @@
 tempArray = [1, 2, 6, 9, 3, 21]
 
 # your ruby loop here
-
+tempArray.map { |element|p element* 2 }
 
 #### 2. From all the built in Ruby methods we've seen in class this week, choose three that you think are particularly helpful and create examples to show how they work.
 
 # your three built in ruby methods
 
+ def add(a, b)
+  a + b
+end
+p add 4,5
+
+
+ x = 3
+  # => 5
+x.times do
+    puts 'Bye'
+    end
+    
+ words = "I like pancakes"
+p words.upcase
+
```

Comment:
You're using .upcase and a .times loop, but can you think of a couple other built in methods in Ruby?